libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -98,6 +100,13 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_YAW_BEHAVE", 8, AP_Follow, _yaw_behave, 1),
 
+    // @Param: _ALT_SOURCE
+    // @DisplayName: Follow altitude source
+    // @Description: Follow altitude source
+    // @Values: 0:absolute, 1: relative
+    // @User: Standard
+    AP_GROUPINFO("_ALT_SOURCE", 9, AP_Follow, _alt_source, AP_FOLLOW_ALTITUDE_SOURCE_RELATIVE),
```

Comment:
_ALT_SOURCE should be moved to the bottom of the parameter list and the "9" conflicts with another existing parameter.  Maybe we should rename it to "_ALT_TYPE" as well.